tests/system/data/test_system_autogen.py

```diff
@@ -0,0 +1,828 @@
+# Copyright 2023 Google LLC
```

Comment:
Yeah, I had the code generator copy over all header comments from the async files. Using a different year for the generated files would be tricky. And I thought since 90% of the files are the same, it makes sense to keep the same publication year, since it's closer to a copy than an original file. But maybe we should use a year range instead?

Reply:
Can we change all 2023 to 2024 for now?